Code/Tools/ProjectManager/Source/GemCatalog/GemCatalogScreen.cpp

```diff
@@ -694,6 +706,18 @@ namespace O3DE::ProjectManager
         }
     }
 
+    void GemCatalogScreen::HandleGemCreated(const GemInfo& gemInfo)
+    {
+        // This signal occurs only upon successful completion of creating a gem. As such, the gemInfo data is assumed to be valid.
+
+        // make sure the project gem catalog model is updated
+        AddToGemModel(gemInfo);
+
+        // create Toast Notification for project gem catalog
+        QString notification = gemInfo.m_displayName + tr(" has been created.");
```

Comment:
could you do this as apart of an argument `tr("%1 has been created.").arg(gemInfo.m_displayName)"`